pom.xml

```diff
@@ -145,7 +145,7 @@
     <properties>
         <siddhi.version>5.1.13</siddhi.version>
         <siddhi.version.range>[5.0.0,6.0.0)</siddhi.version.range>
-        <wso2.transport.file.version>6.0.65</wso2.transport.file.version>
+        <wso2.transport.file.version>6.0.66-SNAPSHOT</wso2.transport.file.version>
```

Comment:
Why have we used a snapshot version here?

Reply:
fixed in https://github.com/siddhi-io/siddhi-io-file/pull/98/commits/4ed5c319c841473322f86c10c2198be28a85b52f